src/js/components/dashboard/table/DashboardTable.jsx

```diff
@@ -55,7 +60,7 @@ const tableHeaders = [
 
 export default class DashboardTable extends React.Component {
     render() {
-        const tableRows = this.props.results.map((row) => (
+        let tableRows = this.props.results.map((row) => (
```

Comment:
Potentially set this to the loading message initially (since that is the how the container state starts), and attempt to do the mapping after `inFlight` is false